test/domain based tests/chainportResolveReport.js

```diff
@@ -0,0 +1,810 @@
+                await env.lssGovernance.connect(adr.member3).committeeMemberVote(2, true);
+                await env.lssGovernance.connect(adr.member4).committeeMemberVote(2, false);
+              });
+
+              it('should resolve negatively', async () => {
+                await env.lssGovernance.connect(adr.lssAdmin).resolveReport(2);
+
+                expect(
+                  await env.lssGovernance.isReportSolved(2),
+                ).to.be.equal(true);
+
+                expect(
+                  await env.lssGovernance.reportResolution(2),
+                ).to.be.equal(false);
+              });
+            });
+
+            describe('when everyone votes positive', () => {
+              beforeEach(async () => {
+                await env.lssGovernance.connect(adr.lssAdmin).losslessVote(2, true);
+                await env.lssGovernance.connect(adr.lerc20Admin).tokenOwnersVote(2, true);
+                await env.lssGovernance.connect(adr.member1).committeeMemberVote(2, true);
+                await env.lssGovernance.connect(adr.member2).committeeMemberVote(2, true);
+                await env.lssGovernance.connect(adr.member3).committeeMemberVote(2, true);
+                await env.lssGovernance.connect(adr.member4).committeeMemberVote(2, true);
+              });
+
+              it('should resolve positively', async () => {
+                await env.lssGovernance.connect(adr.lssAdmin).resolveReport(2);
+
+                expect(
+                  await env.lssGovernance.isReportSolved(2),
+                ).to.be.equal(true);
+
+                expect(
+                  await env.lssGovernance.reportResolution(2),
+                ).to.be.equal(true);
+              });
+            });
+
+            describe('when everyone votes negatively', () => {
+              beforeEach(async () => {
+                await env.lssGovernance.connect(adr.lssAdmin).losslessVote(2, false);
+                await env.lssGovernance.connect(adr.lerc20Admin).tokenOwnersVote(2, false);
+                await env.lssGovernance.connect(adr.member1).committeeMemberVote(2, false);
+                await env.lssGovernance.connect(adr.member2).committeeMemberVote(2, false);
+                await env.lssGovernance.connect(adr.member3).committeeMemberVote(2, false);
+                await env.lssGovernance.connect(adr.member4).committeeMemberVote(2, false);
+              });
+
+              it('should resolve negatively', async () => {
+                await env.lssGovernance.connect(adr.lssAdmin).resolveReport(2);
+
+                expect(
+                  await env.lssGovernance.isReportSolved(2),
+                ).to.be.equal(true);
+
+                expect(
+                  await env.lssGovernance.reportResolution(2),
+                ).to.be.equal(false);
+              });
+            });
+          });
+        });
+      });
+    });
+  });
+});
```

Comment:
Maybe we could add a few test cases to see if fund cannot be moved out with the chainport confirmation.

Reply:
Added more relevant tests when resolving positively. Removed not relevant Report 2 test cases in `chaiportResolveReport.js`